backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/IInvoicingParams.java

```diff
@@ -100,6 +101,8 @@
 	 */
 	boolean isUpdateLocationAndContactForInvoice();
 
+	int getAD_Org_ID();
```

Comment:
use OrgId instead of int. also, i'm not sure this belongs in here. where is it set? the getter seems to be used only in `C_Invoice_Candidate_EnqueueSelectionForInvoicingAndPDFPrinting`